File: week2/day4/week2-day4-workshop/week2-day4-workshop/Sponsor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace week2_day4_workshop
{
    class Sponsor:Person
    {
    public string Company { get; set; }
    public  int HiredStudents { get; set; }

        public Sponsor(string name,int age,string gender, string company)
        {
            this.Name = name;
            this.Age = age;
            this.Gender = gender;
            this.Company = company;
            this.HiredStudents = 0;
        }

        public Sponsor():base("Jane Doe",33,"Female")
        {
            this.Company = "Google";
            this.HiredStudents = 0;
        }

        public void Hire()
        {
            this.HiredStudents++;
        }

        public override void Introduce()
        {
            Console.WriteLine($"Hi, I'm {this.Name}, a {this.Age} year old {this.Gender} who represents {this.Company} and hired {this.HiredStudents} students so far.");
        }
    }
}

File: DSA-2019/week-02/day-1/sponsor.py
from person import Person


class Sponsor(Person):
    def __init__(self, name="Jane Doe", age="30", gender="female", company="Google", hired_students=0):
        Person.__init__(self, name, age, gender)
        self.company = company
        self.hired_students = 0

    def get_goal(self):
        return "Hire brilliant junior software developers."

    def hire(self):
        self.hired_students += 1

    def introduce(self):
        return f"Hi, I'm {self.name}, a {self.age} year old {self.gender} who represents {self.company} and hired {self.hired_students} students so far."
